packages/framework/ini-files/config-specs.ini

```diff
@@ -1732,11 +1748,38 @@ use RHEL7_POST
 
 opt-set-cmake-var CMAKE_CXX_STANDARD STRING FORCE : 17
 
-[rhel7_sems-clang-11.0.1-openmpi-1.10.1-serial_release-debug_shared_no-kokkos-arch_no-asan_no-complex_no-fpic_mpi_no-pt_no-rdc_no-uvm_deprecated-on_all]
```

Comment:
Getting rid of this will break the autotester if we don't make a simultaneous change in there I believe.  Could we leave it, then change the autotester, then delete it?

Reply:
yes thanks for catching that

Reply:
@sebrowne done